skimage/segmentation/tests/test_random_walker.py

```diff
@@ -15,8 +15,7 @@
 PYAMG_MISSING_WARNING = r'pyamg|\A\Z'
 PYAMG_OR_SCIPY_WARNING = SCIPY_RANK_WARNING + '|' + PYAMG_MISSING_WARNING
 
-if (Version(np.__version__) >= '1.15.0' and
-        Version(scipy.__version__) <= '1.1.0'):
```

Comment:
Why is this line removed? Did scipy 1.1.0 change?

Reply:
It should probably be set to < 1.2
https://scipy.github.io/devdocs/release.1.2.0.html